### ISSUE TITLE
No commands found when restarting tests automatically

### ISSUE THREAD

**Issue by xxluke** [cypress-io/cypress#28648]
### Current behavior

When saving edited files (tests) in the cypress folder, the currently running tests are restarted, but stay in loading state as "No commands were issued in this test.".

See this recording: The video starts with showing that the example test usually works. I add a space to the end of the line and save the file, afterwards the test doesn't start anymore.
https://github.com/cypress-io/cypress/assets/8194761/b16ddf29-36bf-4394-9ed7-738b9fcaff38

### Desired behavior

The tests should run, as they used to in the previous versions of Cypress (e.g. 13.5.1).

### Test code to reproduce

I just made a new empty npm directory, installed cypress, and used the Cypress UI to generate the base code.

### Cypress Version

13.6.2

### Node version

v18.17.0

### Operating System

Windows 11 22H2

**Comment by chrisbreiding** [comment]
This might be the same issue as https://github.com/cypress-io/cypress/issues/28392. Does enabling the extension as [detailed here](https://github.com/cypress-io/cypress/issues/28392#issuecomment-1852735772) fix the problem?

**Comment by xxluke** [comment]
Thank you, this is indeed the same issue. For me it's also because of the corporate policy (and unfixed yet).

**Comment by masaid2244** [comment]
I came across an issue where Cypress displayed the following error: 

**The following error originated from your test code, not from Cypress.
> process is not defined
When Cypress detects uncaught errors originating from your test code it will automatically fail the current test.
Cypress could not associate this error to any specific test.
We dynamically generated a new test to display this failure.**

The issue arises when I was working with custom commands and I added one custom command in commands. js file in support, after hours of time wasting on R&D, I just pasted the old commands. js file and all my test cases were working fine.
